src/ResourceManager/AzureStackAdmin/Commands.AzureStackAdmin.Test/App.config

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <appSettings>
+    <add key="AzureStackMachineName" value="azstackvm"/>
```

Comment:
So, I'm going to allow this, but I highly advise against it.  This will make runnign your tests impossible for contributors